app/controllers/users_controller.rb

```diff
@@ -20,10 +28,48 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @user.update(user_params)
```

Comment:
remove ()
what is the difference between update method and update_attributes method

Reply:
- update_attributes method sẽ update tất cả attributes được truyền vào nếu pass validate
- update method sẽ update tất cả các attributes của record chứa id tương ứng, có thể update được nhiều hơn 1 record, và có 2 tham số truyền vào là id và attributes, không check validate